sources/US/SD/school-land.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name" : "SD School Lands",
```

Comment:
Is this needed? If all state school trust lands are open for hunting, and this data is already in the public land layer then i dont think we need it, but i dont know if either of those things is true.

Reply:
I didn't cross reference the Public Land layer. School land is already in there.